tests/SprykerEcoTest/Zed/ComputopShipment/Business/ComputopShipmentFacadeTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\ComputopShipment\Business;
+
+use Codeception\Test\Unit;
+use SprykerEco\Shared\ComputopShipment\ComputopShipmentConstants;
+use SprykerEco\Zed\ComputopShipment\Business\ComputopShipmentException;
+
+class ComputopShipmentFacadeTest extends Unit
+{
+    /**
+     * @var string
+     */
+    public const CORRECT_SHIPMENT_METHOD_KEY = 'spryker_dummy_shipment-standard';
+
+    /**
+     * @var string
+     */
+    public const INCORRECT_SHIPMENT_METHOD_KEY = 'incorrect';
+
+    /**
+     * @var \SprykerEcoTest\Zed\ComputopShipment\ComputopShipmentBusinessTester
+     */
+    protected $tester;
+
+    /**
+     * @var \SprykerTest\Shared\Quote\Helper\PersistentQuoteHelper
+     */
+    protected $persistentQuoteHelper;
+
+    /**
+     * @return void
+     */
+    public function testSuccessExpandQuoteWithDefaultShippingMethod(): void
```

Comment:
please name methods using pattern
test - method name - details

like `testExpandQuoteWithDefaultShippingMethodSuccessfullyExpandsQuote`
same for other similar cases

Reply:
the other case is to have tester class per facade method

would be `ExpandQuoteWithDefaultShippingMethodTest` in this case
and method could be simply named as `testSuccessfullyExpandsQuote` (without facade method prefix)